crates/eigen-cli/src/lib.rs

```diff
@@ -85,6 +124,90 @@ pub fn execute_egnkey_subcommand(subcommand: EigenKeyCommand) -> Result<(), Eige
             println!("{}", operator_id);
             Ok(())
         }
+
+        EigenKeyCommand::BlsConvert {
+            key_type,
+            secret_key,
+            output_path,
+            password,
+        } => {
+            BlsKeystore::from(key_type).new_keystore(
+                secret_key,
+                output_path,
+                password.as_deref(),
+            )?;
+            Ok(())
+        }
+        EigenKeyCommand::CreateNewMnemonic { language, path } => {
+            let mnemonic = match language {
+                MnemonicLanguage::English => {
```

Comment:
if MnemonicLanguage was built with 1) the name string and 2) the word list, this whole match would be unecessary, I think.

Reply:
i think one match will be required, even if we remove it from here, since we have to verify if the name string entered is supported by us.

Reply:
and if they choose to use our word list, i. e without path , then it is absolutely necessary to know the language to use from our list

Reply:
couldn't you could have a TryFrom (with a match, yes) that returns a:
```
pub enum MnemonicLanguage {
  English(&str, &str) // First one the name of the language, second one the work list. might have to be String
}
```
then you'd only have to have one if/then/else, I think

Reply:
the solution i can think of is creating 2 subcommands, one with the user's word list , where we don't use match
and one with our word list, where we use match

Shoudl i do it?

Reply:
just trying to avoid us having to change lot of code when/if adding languages

Reply:
made some changes

Reply:
made some changes!